src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -600,6 +600,9 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                        s.highlightInlineWhispers);
     layout.addCheckbox("Load message history on connect",
                        s.loadTwitchMessageHistoryOnConnect);
+    // TODO: Change phrasing to use better english once we can tag settings, right now it's kept as history instead of historical so that the setting shows up when the user searches for history
+    layout.addIntInput("Max number of history messages to load on connect",
+                       s.twitchMessageHistoryLimit, 10, 800, 10);
```

Comment:
@pajlada, Why is the maximum limit only `800`?

Reply:
IIRC this is how much randers' service returns at max, so there's no need for a higher limit.
Also, loading ~950 or more messages crashes Chatterino, see #2226

Reply:
> 
> 
> IIRC this is how much randers' service returns at max, so there's no need for a higher limit.
> Also, loading ~950 or more messages crashes Chatterino, see #2226

you meant #1421